File: API/Features/Reservations/PickupPoints/ViewModels/PickupPointListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Reservations.PickupPoints {

    public class PickupPointListVM {

        public int Id { get; set; }
        public string Description { get; set; }
        public PickupPointListCoachRouteVM CoachRoute { get; set; }
        public SimpleEntity Destination { get; set; }
        public PickupPointListPortVM Port { get; set; }
        public string ExactPoint { get; set; }
        public string Time { get; set; }
        public string Remarks { get; set; }
        public bool IsActive { get; set; }
        public string PutAt { get; set; }

    }

}

File: FrontEnd/src/app/features/reservations/pickupPoints/classes/view-models/pickupPoint-list-vm.ts
import { PickupPointListCoachRouteVM } from './pickupPoint-list-coachRoute-vm'
import { PickupPointListDestinationVM } from './pickupPoint-list-destination-vm'
import { PickupPointListPortVM } from './pickupPoint-list-port-vm'

export interface PickupPointListVM {

    id: number
    description: string
    coachRoute: PickupPointListCoachRouteVM
    destination: PickupPointListDestinationVM
    port: PickupPointListPortVM
    exactPoint: string
    time: string
    remarks: string
    putAt: string
    isActive: boolean

}
